tests/workflow/eden.workflow.tests.txt

```diff
@@ -1,5 +1,6 @@
 {{$tests := 17}}
 {{$workflow := EdenGetEnv "EDEN_TEST"}}
+eden+ports.sh 2223:2223 2224:2224
```

Comment:
Is it expected behavior to stop-start EVE inside `eden+ports` here before setup of it in case of the medium test (lines below)?

Reply:
WIP